src/app/mat-select-search/mat-select-search.component.scss

```
@@ -30,7 +30,7 @@ $mat-select-panel-padding: 8px;
     TODO: implement proper theming (https://github.com/bithost-gmbh/ngx-mat-select-search/issues/34)
   */
   box-shadow: none;
-  background-color: var(--mat-select-panel-background-color);
+  background-color: var(--mat-sys-surface-container);
```

would it be possible to do a fallback, i.e. something like

```suggestion  
  background-color: var(--mat-select-panel-background-color);
  background-color: var(--mat-sys-surface-container);
```
?

---

well I think not like that.. Maybe: `background-color: var(--mat-sys-surface-container, var(--mat-select-panel-background-color));`?

https://developer.mozilla.org/en-US/docs/Web/CSS/var#using_a_custom_property_as_a_fallback